esmvalcore/preprocessor/_regrid_esmpy.py

```diff
@@ -1,34 +1,41 @@
 # -*- coding: utf-8 -*-
 """Provides regridding for irregular grids."""
 
-import ESMF
+try:
+    import esmpy
+except ImportError as exc:
+    # Prior to v8.4.0, `esmpy`` could be imported as `ESMF`.
+    try:
+        import ESMF as esmpy  # noqa: N811
+    except ImportError:
+        raise exc
```

Comment:
Is this `try` `except` necessary?

Reply:
I thought it might be nicest to raise the original exception, but I could remove it if you think it's not needed.